tests/integration/pure_cairo_vm/cairo1_integration/test_cairo1.cairo

```diff
@@ -0,0 +1,4 @@
+#[test]
```

Comment:
Would be nice to include a dependency library for this test

Reply:
Would be nice to also test cases
- multiple suites
- no tests found

Reply:
[Add a test with custom library](https://github.com/software-mansion/protostar/pull/1522/commits/74d25112738510bc020275f2c2da64ee15a3d2da)
[Add empty case + combined case with 2 suites](https://github.com/software-mansion/protostar/pull/1522/commits/a129018ca163ac5fd55e2ceb69dcc411c565d42b)